lib/services/snap.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
       service : 'snap',
       build : env.SNAP_PIPELINE_COUNTER,
       commit : env.SNAP_COMMIT,
-      branch : env.SNAP_BRANCH,
+      branch : env.SNAP_BRANCH || env.SNAP_UPSTREAM_BRANCH,
```

Comment:
Yes. I was thinking the same on my way home. I was wondering if we should
use the pull request number instead of the upstream branch? Happy to change
the PR if you both think so.

- Sent from /dev/mobile
Akshay Karle
On Jun 30, 2015 7:34 PM, "Kent C. Dodds" <notifications@github.com> wrote:

> In lib/services/snap.js
> <https://github.com/cainus/codecov.io/pull/18#discussion_r33631743>:
>
> > @@ -11,7 +11,7 @@ module.exports = {
> >        service : 'snap',
> >        build : env.SNAP_PIPELINE_COUNTER,
> >        commit : env.SNAP_COMMIT,
> > -      branch : env.SNAP_BRANCH,
> > +      branch : env.SNAP_BRANCH || env.SNAP_UPSTREAM_BRANCH,
>
> on a PR to master, this would resolve to be master, which would result (if
> I'm not mistaken) in the coverage reports to override what is in master and
> I don't believe that's what we want. I may be simply misunderstanding how
> codecov works though...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cainus/codecov.io/pull/18/files#r33631743>.
>